projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -237,6 +237,10 @@ public function add_new_site_link() {
 	 * Adds site card component.
 	 */
 	public function add_site_card_menu() {
+		// When the interface is set to wp-admin, we not add the site_card.
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
```

Comment:
Not important, but wondering what's the norm for the jetpack repo
`'wp-admin' === get_option( 'wpcom_admin_interface' )` or `get_option( 'wpcom_admin_interface' ) === 'wp-admin'`

Reply:
Yep, I think it is, I'll update all my PRs